### PR TITLE
Improve post-processor GUI and service

### PR DESCRIPTION
**Pull request by VictorLamoine**
Improve / finish GUI

Program location is now set by a fixed value: 
https://github.com/InstitutMaupertuis/fanuc_grinding/pull/47/files#diff-2cf2a9843953106c17cd22e22b8834a1R221

Add `sendMsgBox` in the widget

Check service return status

rename `getRobotPosesData` to `getRobotTrajectoryData`

### CODE REVIEW

**Comment by VictorLamoine** [comment]
@KevinBollore ready for review

File: grinding_rviz_plugin/src/grinding_rviz_plugin.cpp [hunk 1]
```diff
@@ -67,7 +68,7 @@ GrindingRvizPlugin::GrindingRvizPlugin(QWidget* parent) :
   // SCANNING
   // Will display a status in general status label ( from scanning widget )
   connect(scanning_widget_, SIGNAL(sendStatus(QString)), this, SLOT(displayStatusHandler(QString)));
-  connect(scanning_widget_, SIGNAL(sendMsgBox(QString, QString , QString)),
+  connect(scanning_widget_, SIGNAL(sendMsgBox(QString, QString, QString)),
```
Comment (KevinBollore) [inline grinding_rviz_plugin/src/grinding_rviz_plugin.cpp]: Not concerned by this PR, but not a big deal.

File: grinding_rviz_plugin/src/grinding_rviz_plugin.cpp [hunk 2]
```diff
@@ -137,12 +138,14 @@ GrindingRvizPlugin::GrindingRvizPlugin(QWidget* parent) :
   //POST_PROCESSOR
   // Will display a status in general status label ( from post_processor widget )
   connect(post_processor_widget_, SIGNAL(sendStatus(QString)), this, SLOT(displayStatusHandler(QString)));
+  connect(post_processor_widget_, SIGNAL(sendMsgBox(QString, QString, QString)),
+                      this, SLOT(displayMsgBoxHandler(QString, QString, QString)));
```
Comment (KevinBollore) [inline grinding_rviz_plugin/src/grinding_rviz_plugin.cpp]: To my mind, It is more readable like this.
```
connect(post_processor_widget_, SIGNAL(sendMsgBox(QString, QString, QString)),
                          this, SLOT(displayMsgBoxHandler(QString, QString, QString)));
```
For long lines, I tried to write all the `connect()` like this, except mistakes.

**Comment by KevinBollore** [comment]
Just cosmetic, but it seems strange that the checkbox is on the middle of the widget. Maybe we can use a stretch to put it little bit more on left ( near of the label )? Only my mind, as you want.
image

File: grinding_rviz_plugin/src/grinding_rviz_plugin.cpp (hunk 2 above)
Comment (VictorLamoine) [inline grinding_rviz_plugin/src/grinding_rviz_plugin.cpp]: I can't see the change in your example

Comment (KevinBollore) [inline grinding_rviz_plugin/src/grinding_rviz_plugin.cpp]: Please see line 124 & 125 in the same file for a similar example.

Comment (VictorLamoine) [inline grinding_rviz_plugin/src/grinding_rviz_plugin.cpp]: I'll make an other PR and Eclipse format all this paragraph.

**Comment by VictorLamoine** [comment]
All fixed, ok to merge?

**Comment by KevinBollore** [comment]
Yes

**Pull request merged**